src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Styles/GroupBox.xaml

```diff
@@ -0,0 +1,51 @@
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:system="clr-namespace:System;assembly=System.Runtime">
+
+    <Thickness x:Key="GroupBoxPadding">0,0,0,16</Thickness>
+    <system:Double x:Key="GroupBoxHeaderFontSize">20</system:Double>
+    <Thickness x:Key="GroupBoxHeaderMargin">0,4,0,12</Thickness>
+    <Thickness x:Key="GroupBoxBorderThickness">0</Thickness>
+
+    <Style x:Key="DefaultGroupBoxStyle" TargetType="{x:Type GroupBox}">
+        <Setter Property="OverridesDefaultStyle" Value="True" />
+        <Setter Property="BorderThickness" Value="{DynamicResource GroupBoxBorderThickness}" />
+        <Setter Property="Background" Value="{DynamicResource GroupBoxBackground}" />
+        <Setter Property="BorderBrush" Value="{DynamicResource GroupBoxBorderBrush}" />
+        <Setter Property="Foreground" Value="{DynamicResource GroupBoxForeground}" />
+        <Setter Property="Padding" Value="{StaticResource GroupBoxPadding}" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="{x:Type GroupBox}">
+                    <Border
+                        BorderBrush="{TemplateBinding BorderBrush}"
+                        BorderThickness="{TemplateBinding BorderThickness}"
+                        Background="{TemplateBinding Background}"
+                        SnapsToDevicePixels="True">
+                        <Grid>
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="Auto" />
+                                <RowDefinition Height="*" />
+                            </Grid.RowDefinitions>
+                            <ContentPresenter
+                                Grid.Row="0"
+                                TextElement.FontSize="{DynamicResource GroupBoxHeaderFontSize}"
+                                Margin="{DynamicResource GroupBoxHeaderMargin}"
+                                ContentSource="Header"
```

Comment:
We should do a DynamicResource binding of Header ContentPresenter's Foreground to a resource. 

Better to convert GroupBoxForeground -> GroupBoxHeaderForeground and use here.